crates/tokio-util/src/event_listeners.rs

```diff
@@ -1,46 +1,59 @@
-use tokio::sync::mpsc;
-use tokio_stream::wrappers::UnboundedReceiverStream;
+use tokio::sync::broadcast::{self, Sender};
+use tokio_stream::wrappers::BroadcastStream;
+use tracing::{error, warn};
 
-/// A collection of event listeners for a task.
-#[derive(Clone, Debug)]
+const DEFAULT_BROADCAST_CHANNEL_SIZE: usize = 1000;
```

Comment:
for constant names it's nice if we can adhere to this grammar roughly https://github.com/paradigmxyz/reth/issues/6464
```suggestion
const DEFAULT_SIZE_BROADCAST_CHANNEL: usize = 1000;
```
or even `DEFAULT_MAX_COUNT_MSGS_BROADCAST_CHANNEL`